uber/templates/preregistration/dealer_registration.html

```diff
@@ -1,10 +0,0 @@
-{% extends "./preregistration/preregbase.html" %}
-{% block title %}Preregistration{% endblock %}
-{% block backlink %}{% endblock %}
-{% block content %}
-
```

Comment:
This template was added last year for MAGFest's purposes - it lets people send out a user-friendly link for dealer registration. Can we add it back in, or use another solution?